src/components/common/Form/Form.scss

```diff
@@ -0,0 +1,28 @@
+@import '../../../assets/css/colors';
+
+#Form {
+  #formTitle {
+    background: $yellow;
+    padding: 15px;
+    margin-top: -55px !important;
+    margin-left: -30px !important;
+    margin-right: -30px !important;
+    margin-bottom: 30px !important;
```

Comment:
for pixels, I suggest that you import `pixels.scss` file here and use variables from it